### PR TITLE
New empty-state and left panel bugfix

### LINKED ISSUE #2968: UI bug: LeftPanel overlaps canvas
With the recent UI changes to the left panel / canvas we introduced a small UI bug where the left panel hides the canvas behind it, rendering timelines inaccessible (see screenshot below). The issue happens only with empty sketches and after uploading the first timeline via the web UI. It is not blocking, since a reload of the page fixes it permanently.

**How to reproduce**
1. Create a blank sketch via the web UI
2. Upload a timeline to this sketch via the web UI
3. See the status observed in the screenshot below.

**Screenshots**
image

### LINKED ISSUE #2992: Update the Sketch is archived page
Follow-up issue to PR #2991 :
* Update the "This sketch is archived" page to the same / similar look as the "It's empty around here" page.

### PR DESCRIPTION
**Pull request by berggren**
This PR fixes a bug where the left navigation drawer overlapped the canvas main section. A couple of other fixes:
- Reworked the empty state page to vertically align content and add art
- Removed old unused (and not functional) resize panel code
- Use v-model for show/hide panel instead of width

 
Screenshot 2023-11-22 at 14 02 38

**Closing issues**
closes #2968 
closes #2992

### CODE REVIEW

File: timesketch/frontend-ng/src/views/Sketch.vue [hunk 1]
```diff
@@ -208,13 +211,13 @@ limitations under the License.
 
       <!-- Left panel -->
       <v-navigation-drawer
-        v-show="showLeftPanel && hasTimelines && !isArchived"
+        v-model="showLeftPanel"
```
Comment (jkppr) [inline timesketch/frontend-ng/src/views/Sketch.vue:214]: This change will allow to have an opened left panel on an archived sketch. Do we want/need to have the left panel active on an archived sketch?

This results in an odd view like this:
image

File: timesketch/frontend-ng/src/views/Sketch.vue [hunk 2]
```diff
@@ -435,11 +438,9 @@ export default {
       this.$store.dispatch('updateContextLinks')
       this.$store.dispatch('updateAnalyzerList', this.sketchId)
       this.loadingSketch = false
-      this.showLeftPanel = true
-      this.$nextTick(function () {
-        this.setDrawerBorderStyle()
-        this.setDrawerResizeEvents()
-      })
+      if (this.hasTimelines) {
```
Comment (jkppr) [inline timesketch/frontend-ng/src/views/Sketch.vue:441]:
```suggestion
      if (this.hasTimelines && !this.isArchived) {
```

To close the left panel on archived sketches per default. (Still allows them to be opened on an archived sketch though)

**Review by jkppr — CHANGES_REQUESTED** [review]
Two small nits below

**Comment by jkppr** [comment]
As discussed, added an issue to track the follow-up task for the "archived sketch" page: #2992

File: timesketch/frontend-ng/src/views/Sketch.vue (hunk 1 above)
Comment (berggren) [reply timesketch/frontend-ng/src/views/Sketch.vue:214]: Good catch. Fixed now. Same behaviour and design for the archived page.

**Comment by berggren** [comment]
> As discussed, added an issue to track the follow-up task for the "archived sketch" page: #2992

I ended up fixing it in this PR. PTAL

(the unit test fail seem to be a temporary issue with the apt mirror)

**Review by jkppr — APPROVED** [review]
lgtm

**Pull request merged**